### PR TITLE
version switching fixes

### PR DESCRIPTION
**Pull request by toddjordan**

### CODE REVIEW

File: app/routes/project-version.js [hunk 1]
```diff
@@ -68,17 +68,19 @@ export default Route.extend({
           endingRoute = `namespaces/${namespaceName}`;
           break;
         }
+        case 'project-version.classes.class.methods.method': {
```
Comment (toddjordan) [inline app/routes/project-version.js]: actually need to add 2 more cases for events in and properties.

Comment (jenweber) [reply app/routes/project-version.js]: I'm also seeing that being on the methods tab blocks route changes. Example: http://localhost:4200/ember/2.14/classes/Ember.Route/methods

Comment (toddjordan) [reply app/routes/project-version.js]: @jenweber unsure what you are referring to: 
https://emberjs.com/api/ember/2.15/classes/Ember.Component/methods
https://emberjs.com/api/ember/2.14/classes/Ember.Route/methods

Comment (toddjordan) [reply app/routes/project-version.js]: ok. maybe I do. Do you mean if I am on https://emberjs.com/api/ember/2.14/classes/Ember.Route/methods and switch versions, it resets to application module and not to the methods route of the new version?  Fixing that as well.

Comment (jenweber) [reply app/routes/project-version.js]: @toddjordan I mean that when I was on the methods tab, if I tried to change versions, the version route change never happened at all. The dropdown and url indicated that I was still on the same version. However version switching worked when I was on the index tab.

**Comment by toddjordan** [comment]
Ok. Should be fixed by the last update. I’m planning on adding some tests
as well

**Comment by toddjordan** [comment]
@jenweber fixed up version switching and added tests

**Pull request merged**